kivy/config.py

```diff
@@ -842,6 +842,9 @@ def name(self, value):
         elif version == 18:
             Config.setdefault('kivy', 'log_maxfiles', '100')
 
+        elif version == 19:
+            Config.setdefault('kivy', 'dispatch_cursor_movement', '0')
```

Comment:
This probably belongs in the `postproc` section.

Reply:
Good idea, should have done before, is updated now.